routes/api.js

```diff
@@ -1,10 +1,28 @@
 const express = require('express');
 const router = express.Router();
 
+const models = require('../database/models');
+
+const localUser = models.user_account;
 /* GET api listing. */
 router.get('/getUserInfo', (req, res) => {
     res.status(200).json({user: req.user});
 });
 
-module.exports = router;
+router.get('/username', (req, res) => {
+    const email=req.user.email;
```

Comment:
spacing

how is this different from the one in https://github.com/systers/PC-Prep-Kit/pull/21/files?